frontend/packages/core/src/AppLayout/notifcations.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "@emotion/styled";
+import {
+  Box,
+  ClickAwayListener,
+  Grow,
+  IconButton,
+  MenuList,
+  Paper,
+  Popper,
+} from "@material-ui/core";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+
+const StyledNotificationsIcon = styled(IconButton)`
+  color: #ffffff;
+  margin-right: 1rem;
```

Comment:
I asked Kanchan for the hover and pressed states

Reply:
I added the notfication states. Figma designs set the padding to 12px but it looked too large on the header. Example:
<img width="1911" alt="Screen Shot 2020-12-01 at 2 10 35 PM" src="https://user-images.githubusercontent.com/39421794/100785683-00dae080-33df-11eb-9d89-4b360c98193b.png">

So I brought it down to 8px.